frontend/packages/dev-console/src/components/builds/BuildsTabListPage.tsx

```diff
@@ -96,13 +96,18 @@ const BuildsTabListPage: React.FC<BuildsTabListPageProps> = ({ match }) => {
     return (
       <CreateProjectListPage title={title}>
         {(openProjectModal) => (
-          <Trans t={t} ns="devconsole">
-            Select a Project to view the list of builds or{' '}
-            <Button isInline variant="link" onClick={openProjectModal}>
-              create a Project
-            </Button>
+          <>
+            {t('devconsole~Select a Project to view the list of builds')}{' '}
+            {canCreateProject && (
+              <Trans t={t} ns="devconsole">
+                or{' '}
+                <Button isInline variant="link" onClick={openProjectModal}>
+                  create a Project
+                </Button>
+              </Trans>
+            )}
```

Comment:
I believe you missed using your new component here, or is there a difference that I didn't see?